eng/common/docgeneration/Generate-DocIndex.ps1

```diff
@@ -119,7 +119,9 @@ function GenerateDocfxTocContent([Hashtable]$tocContent, [String]$lang, [String]
         $serviceName = $serviceMapping.Value[0]
         $displayName = $serviceMapping.Value[1]
 
-        $fileName = ($serviceName -replace '\s', '').ToLower().Trim()
+        # handle spaces in service name, EG "Confidential Ledger"
+        # handle / in service name, EG "Database for MySQL/PostgreSQL". Leaving a "/" present will generate a bad link location.
+        $fileName = ($serviceName -replace '\s', '').Replace("/","").ToLower().Trim()
```

Comment:
We can combine these two conditions into one regex. 
$serviceName -replace "\s|\\|\/", ""
I am wondering why the sync PR generated?

Reply:
Because I was moving this along!

Reply:
I'm going to merge this Sima. This is just the sync PR anyway. I'll do a follow-up, but I've already merged one of these before your review feedback.